lexibank_tls.py

```diff
@@ -636,10 +638,19 @@ def cmd_makecldf(self, args):
 
         # Add concepts
         concept_lookup = args.writer.add_concepts(
```

Comment:
@chrzyki, you do not need to split the glosses, since the NURSE concept list should have the splitter inside the CSVW, right? This means, you better follow the approach to add each concept in a loop (not using the shortcut, which is less flexible!) and making the lookup at the same time:

```python
concepts = {}
for concept in self.conceptlists[0].concepts.values():
    idx = ...
    for gloss in concept.attributes["lexibank_gloss"]:
        concepts[gloss] = idx
    args.writer.add_concept(
        ID=idx,
        Swaheli_Gloss=concept.attributes["Swaheli"],
        ....
    )
```
I absolutely prefer this solution over alternatives, since it allows you to also add the Swaheli gloss, which would otherwise not be possible, and the Swaheli gloss is important for interpretation here, if you load the CLDF,

Reply:
BTW, update: we do not NEED the lexibank gloss (but we need Swaheli): The concepts are numbered, as we discussed when linking to concepticon, we have 1038, 1038a, etc., and this information is in the original data file. So you can ignore the lexibank gloss and just assemble BY numerical code! That was anyway my intention all the time. So we use the numerical code, and all you have to adjust in the Lookup (please also add the Number to the concept class), is that the source has floating points at times, so you delete the `.0` on some occasions to guarantee it matches.